server/weaverbird/pipeline/steps/custom.py

```diff
@@ -0,0 +1,8 @@
+from pydantic import Field
+
+from weaverbird.pipeline.steps.utils.base import BaseStep
+
+
+class CustomStep(BaseStep):
```

Comment:
For me i suggest `CustomSqlStep` instead of `CustomStep`.... as the class name, because this can be confusing with the customStep alreadyavailable on the frontend but with mongo... i don't know 🤔 same for `CUSTOM_STEP` to `CUSTOMSQL_STEP` ?
What do you think ?

Reply:
Good remark, I update that.